themes/django20/layouts/index.html

```diff
@@ -11,26 +11,27 @@
   <section class="events" id="events">
     <div class="container container-events {{ .Params.cssClassSuffix }}">
       <article class="{{ .Params.cssClassSuffix }}">
-        
-        <h1>Venue events</h1>
-        
-        {{ $events := (where (.Site.GetPage "/events").Pages.ByDate "published" "!=" "false") }}
 
-        <table class="event-table">
+        {{ $events := .Site.Data.events }}
```

Comment:
I see what you did here!

But the events are supposed to be managed as individual files in `/content/events/` - that's how we get the full flexibility. Each event has its own markdown file, but the front matter (the yaml code at the top) has the data in a proper schema.

And no merge conflicts when people are editing the same big JSON soup :smile: